src/vector/platform/ElectronPlatform.tsx

```diff
@@ -367,9 +367,15 @@ export default class ElectronPlatform extends VectorBasePlatform {
             this._ipcCall('focusWindow');
         };
 
+        ipcRenderer.send('startBlinking');
         return notification;
     }
 
+    clearNotification(notif: Notification) {
+        super.clearNotification(notif);
+        ipcRenderer.send('stopBlinking');
```

Comment:
This doesn't really seem like the right place to put these: lots of notifications may be displayed and then some of them may be cleared, so this will cause inconsistent and buggy behaviour, eg. the icon will start blinking whenever the client displays a notification, but will stop blinking if the user reads a message (possibly on a different device), even if they still have other unread messages.

If the intention here is to blink whenever there are unread notifications, why not just make the icon blink if the badge count is nonzero?

Reply:
You're right, I refactored the code and made it use the notification count instead.